content/millennium/r4/medications/medication-request.md

```diff
@@ -139,6 +139,56 @@ We are currently always returning a value of "Order" in the intent field. We rec
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Create
+
+Create an individual MedicationRequest.
+
+    POST /MedicationRequest
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:medication_request, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationRequest
+
+#### Body
+
+<%= json(:r4_medication_request_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Wed, 27 Mar 2019 15:59:33 GMT
+Etag: W/"0"
+Last-Modified: Wed, 27 Mar 2019 15:59:30 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationRequest/4595905
+Server-Response-Time: 3890.363996
+Status: 201 Created
```

Comment:
Please remove Status - I thought we already removed this non-standard from the response, but it causes some confusion. May consider removing others that we aren't purposefully adding for devs to use? (eg: X-Runtime and Server-Response Time?)

Reply:
Removed.

Reply:
Thoughts about the other fields I mentioned?

Reply:
I'm not really sure. Personally, I'd say if it's not useful it's not really worth clogging up the page, but all that is listed on the other resources and I think it should at least be consistent.

Reply:
We caused some confusion when status was removed. Technically, these are created when you make the request (status wouldn't be in the current response). I don't want to document headers in these examples that we're not intended intentionally to produce, in order to avoid the issue we have with status (Status is also listed in other examples, which are now out of date). I always will vote for "document with intent" rather than accident :)

Reply:
I'd say X-Runtime and Server-Response-Time probably should be removed then, but I'm not really 100% which ones we're adding for devs to use 😅

Reply:
fine to remove for this PR and we will loop through other examples later https://github.com/cerner/fhir.cerner.com/issues/397

Reply:
Removed X-Runtime and Server-Response-Time: https://github.com/cerner/fhir.cerner.com/pull/353/commits/5b4b8c951640267e7d7ac2be448cdeae49b511d0 Let me know if you two think others should be removed as well, since like I said I'm not 100% sure.

Reply:
removing just those headers is ok